index.d.ts

```diff
@@ -24,5 +24,6 @@ declare module 'mongo-unit' {
         dbpath: string;
         verbose: boolean;
         version: string;
+        useReplicaSet: boolean;
```

Comment:
it probably should be optional
`useReplicaSet?: boolean;`

Reply:
yes, you are right.
I have updated.